app/models/comfy/cms/block.rb

```diff
@@ -1,59 +1,70 @@
 class Comfy::Cms::Block < ActiveRecord::Base
   self.table_name = 'comfy_cms_blocks'
 
-  FILE_CLASSES = %w(ActionDispatch::Http::UploadedFile Rack::Test::UploadedFile File).freeze
+  FILE_CLASSES = %w[ActionDispatch::Http::UploadedFile
+                    Rack::Test::UploadedFile File].freeze
 
   attr_accessor :temp_files
 
   # -- Relationships --------------------------------------------------------
   belongs_to :blockable,
-    polymorphic: true
+             polymorphic: true
+
   has_many :files,
-    autosave: true,
-    dependent: :destroy
+           autosave: true,
+           dependent: :destroy
 
   # -- Callbacks ------------------------------------------------------------
   before_save :prepare_files
 
   # -- Validations ----------------------------------------------------------
-  validates :identifier, :presence => true
+  validates :identifier, presence: true
 
   # -- Instance Methods -----------------------------------------------------
   # Tag object that is using this block
   def tag
-    @tag ||= blockable.tags(:reload).detect{|t| t.is_cms_block? && t.identifier == identifier}
+    @tag ||= blockable.tags(:reload).detect do |t|
+      t.is_cms_block? && t.identifier == identifier
+    end
   end
 
   # Intercepting assigns as we can't cram files into content directly anymore
   def content=(value)
     self.temp_files = [value].flatten.select do |f|
       FILE_CLASSES.member?(f.class.name)
     end
-
     # correctly triggering dirty
-    value = nil               if self.temp_files.present?
-    write_attribute(:content, value)
-    self.content_will_change! if self.temp_files.present?
+    if temp_files.present?
+      write_attribute(:content, nil)
+      content_will_change!
+    else
+      write_attribute(:content, value)
+    end
```

Comment:
Could use a guard clause as well, but I think this is already a huuuge improvement compared to what we had... 🤦‍♂️

Reply:
I personally tend to favour classical `if/else` over guard clauses unless guarding against extreme cases or having multiple levels of conditionals. 
When is a single "do A or B depending on some condition" not falling under any of those cases I prefer the classical if else. But is a totally biased thing :P